### ISSUE TITLE
[SF_ACC][Feature Request][FileUploader]: adding "required" & "accessibleName" property

### ISSUE THREAD

**Issue by ksblklu02** [SAP/ui5-webcomponents#9096]
### Feature Request Description

With JAWS, when user navigate with Tab key to the FileUploader component, If the corresponding label is set to "required", the component cannot announce the "required" information and the field label text.

### Proposed Solution

Adding `requried` & `accessibleName` property to the FileUploader component.

### Priority

Medium

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @ksblklu02 ,

Closing this issue because it is duplicate of https://github.com/SAP/ui5-webcomponents/issues/9095.

Regards,
Plamen Ivanov
